internal/resp_assertions/acl_getuser_response_assertion.go

```diff
@@ -0,0 +1,150 @@
+package resp_assertions
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"slices"
+
+	resp_value "github.com/codecrafters-io/redis-tester/internal/resp/value"
+)
+
+type passwordHashPair struct {
+	Password string
+	Hash     string
+}
+
+type AclGetuserResponseAssertion struct {
+	expectedFlags     []string
```

Comment:
```suggestion
	flagsExpectedToBePresent     []string
	flagsExpectedToBeAbsent      []string
```

(the term "unexpected" is a bit ambiguous)